gruf.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |spec|
   spec.executables << 'gruf'
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.7', '< 3.3'
+  spec.required_ruby_version = '>= 2.7', '< 3.4'
```

Comment:
Is there a reason why the ruby version has an upper bound other than confirming with testing?

Removing this upper bound would avoid this issue where this library prevents applications from updating ruby.

Reply:
Given gRPC's often incompatibilities with new Ruby versions, we've pinned this library explicitly until tests can be done.

We'll get a working build of this up this week, with 3.3 support.

Reply:
Happy new year, @splittingred ! I wonder if there is any update on this issue. 😄